kotlin-remix-run-router/src/main/generated/remix/run/router/BlockerUnblocked.kt

```diff
@@ -0,0 +1,18 @@
+@file:JsModule("@remix-run/router")
+@file:JsNonModule
+
+@file:Suppress(
+    "NON_EXTERNAL_DECLARATION_IN_INAPPROPRIATE_FILE",
+    "NAME_CONTAINS_ILLEGAL_CHARS",
+)
+
+package remix.run.router
+
+
+external interface BlockerUnblocked {
+    var state: String
+    var reset: Any? /* some expression */
```

Comment:
"some expression" can be in comment?

Reply:
Right now "some expression" is rendered when there is no type at all or type is `null` or `undefined`, I need to rework some logic in converter to separate these two cases and then I can write some more useful comments in such places.